magento/magento2
# upgrade tubalmartin/cssmin to 4.1.0

### Description


According to https://github.com/tubalmartin/YUI-CSS-compressor-PHP-port/blob/master/README.md#v410-16-may-2017: Performance: 2x times faster than v4.0.0 after code profiling

Also minor enhancements like:

    [target='_blank']       => [target=_blank]
    [type="something"]      => [type=something]
    background: transparent => background: 0 0
    background: none        => background: 0 0
    #f00                    => red
    padding: 20px 20px      => padding 20px
    0.12                    => .12

total min.css size reduced by 2563 bytes (from 1732225 to 1729662)

### Fixed Issues (if relevant)

1. none as if yet

### Manual testing scenarios

1. Before upgrading this package, generate minified css files, then run ``cat `find pub/static/ -name *.min.css -print | sort` > 3.0.0.min.css ``
2. Upgrade the package, generate minified css again and run ``cat `find pub/static/ -name *.min.css -print | sort` > 4.1.0.min.css ``
3. Compare the two files. you will see the enhancements like above.
4. Visually compare the site before and after and see that there are no changes.

### Contribution checklist
